foundationdb/README.md

```diff
@@ -0,0 +1,149 @@
+# Agent Check: FoundationDB
+
+## Overview
+
+This check monitors [FoundationDB][1] through the Datadog Agent. Aside from
+checking that the FoundationDB cluster is healthy, it also collects numerous metrics
+and, optionally, FoundationDB transaction logs.
+
+## Setup
+
+Both the check and metrics apply to the FoundationDB cluster as a whole,
+and should only be installed on one host. This doesn't need to be one that is
+running FoundationDB, but just one with access to it.
+
+### Installation
+
+The FoundationDB check is included in the [Datadog Agent][2] package,
+but in it requires to have the [FoundationDB client][8] installed.
+
+### Configuration
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Host" xxx -->
+
+#### Host
+
+To configure this check for an Agent running on a host:
```

Comment:
Seems to be missing information here? Or is "Metric collection" and "Log collection" supposed to be under this section?

Reply:
Thanks for letting me know. Looks like "Metric collection" and "Log collection" should be under this section, which they both are now.